pipeline-modules/app-service/aws/docker-build-n-push.yaml

```diff
@@ -42,9 +42,8 @@ template: |
   phases:
     pre_build:
       commands:
-        - echo Logging in to Amazon ECR...
-        - aws --version
-        - $(aws ecr get-login --region $AWS_DEFAULT_REGION --no-include-email)
+        - echo Logging in to docker
+        - echo $docker_password | docker login -u $docker_username --password-stdin $docker_registry
```

Comment:
Have tested this yet but will that echo put the password in plaintext in the pipeline log?

Reply:
Not expected to do so. If I run this on my terminal, the output of echo is directly piped into the next command, and it's not printed on terminal (PFA screenshot).
Also, we've seen that on cloud native build tools, even if we purposely print a sensitive string, they have some mechanism to hide it. AWS replaces it with `***` and I forgot what GCP does. Although, this is just kind of a seat-belt and the first point above should avoid the accident.

![image](https://user-images.githubusercontent.com/17708249/147046495-55dbba22-3847-48b3-b536-29111046b0ff.png)